config/environments/production.rb

```diff
@@ -55,8 +55,8 @@
   # Prepend all log lines with the following tags.
   config.log_tags = [ :request_id ]
 
-  # Use a different cache store in production.
-  # config.cache_store = :mem_cache_store
+  # Use redis as the cache in production.
+  config.cache_store = :redis_cache_store, { url: ENV['REDIS_URL'] }
```

Comment:
```suggestion
  config.cache_store = :redis_cache_store, { url: Settings.REDIS_URL }
```

Is there magic I'm missing, or should we use the railsconfig configuration from over in shared_configs?

Reply:
I saw `ENV['REDIS_URL']` in the sidekiq initializers so figured that was the best thing to do, although I guess I didn't look super hard since it does both of those things:

https://github.com/sul-dlss/earthworks/blob/1a04d54c3bfec0a290deb1f4f00e2707ab5684a4/config/initializers/sidekiq.rb#L1-L7

Should we just do the same here?

Reply:
👍